src/components/Admin/AdminPage.js

```diff
@@ -1,20 +1,24 @@
 import React from 'react';
-import { useHistory, useLocation } from 'react-router-dom';
+import { useHistory } from 'react-router-dom';
+import { connect } from 'react-redux';
+import PropTypes from 'prop-types';
 import { AdminPageButtons } from '../../utils/configs';
 import { PageButton } from '../Buttons';
 import MenuHelper from '../../helpers/MenuHelper';
+import { mapStateToProps } from '../../redux/actions';
 
-const DriverPage = () => {
+const AdminPage = ({ appState }) => {
   const history = useHistory();
-  const location = useLocation();
+  const { userId } = appState;
 
   const onSelect = (event) => {
-    const path = MenuHelper({ event, location });
+    const path = MenuHelper({ event, userId });
     history.push(path);
   };
 
   const buttons = AdminPageButtons.map((button) => (
     <PageButton
+      key={button[0]}
```

Comment:
if you change to object declaration you avoid to get first element of button array. **

Reply:
Changed